src/ensembl/src/shared/components/visibility-icon/VisibilityIcon.scss

```diff
@@ -0,0 +1,56 @@
+@import 'src/styles/common';
+
+.selected {
+  svg {
+    fill: $dark-blue;
+  }
+  &svg:hover {
+    opacity: 0.8;
+  }
+}
+
+.unselected {
+  svg {
+    fill: $grey;
+  }
+  &svg:hover {
+    opacity: 0.8;
+  }
+}
+
+.disabled {
+  cursor: not-allowed;
+  svg {
+    fill: $grey;
+    opacity: 1;
+  }
+  &:before {
+    position: absolute;
+    content: '';
+    top: 50%;
+    border-color: #2c343a;
+    margin-top: -1px;
+    border-top: 2px solid red;
+    transform: rotate(-45deg);
+    width: 100%;
+  }
+  &:after {
+    position: absolute;
+    content: '';
+    left: 0px;
+    border-color: #2c343a;
+    border: 2px solid red;
+    border-radius: 50%;
+    height: 100%;
+    width: 100%;
+  }
+}
```

Comment:
I still think this 🚫 over the disabled visibility icon drawn by the `before` and `after` pseudoelements is entirely unnecessary.

Reply:
Removed after confirming with Andrea. Thanks